radiation/radiation_ecckd.F90

```diff
@@ -456,7 +456,7 @@ end subroutine read_spectral_solar_cycle
   ! at nlev layers
   subroutine calc_optical_depth_ckd_model(this, ncol, nlev, istartcol, iendcol, nmaxgas, &
        &  pressure_hl, temperature_fl, mole_fraction_fl, &
-       &  optical_depth_fl, rayleigh_od_fl, concentration_scaling)
+       &  optical_depth_fl, rayleigh_od_fl, opt_concentration_scaling)
```

Comment:
I would prefer not to change the interface, i.e., keep the name `concentration_scaling` here - the `opt_` prefix is not used elsewhere for optional arguments either.

Reply:
Thanks. Let me know if your local tests were successful, then I can perform this change. Any preferences for the naming of the local variable (currently `concentration_scaling`)?

Reply:
No specific preference, for a local variable we're also not very restricted. Intuitively I might have gone for something like a `local_` prefix, but `opt_` or something else would also be fine.